notebooks/entities-synonyms/embedding_synonyms.py

```diff
@@ -0,0 +1,55 @@
+from src.utils.preprocess import get_n_word_strings
+from src.utils.helper_embedding import get_embedding_synonyms
+
+from os import getenv
+import pandas as pd
+import csv
+import json
+from tqdm import tqdm
+
+
+DATA_DIR = getenv('DATA_DIR')
```

Comment:
This practice of having DATA_DIR seems a bit endemic in ds scripts, it's not a very good practice for the following reasons
1. If you have multiple projects each with their own data_dir (which is different) you have to change it, which is annoying and will cause problems/slow you down
2. There's no reason for it to be an environment variable, it's not a secret
3. It's (nearly) the same folder as lines 13 and 14 below, it's unnecessarily confusing for them to be different
4. The folder is in the same project so its ok to assume where it'll be, hard coding in "data/foo.bar" is ok

Reply:
Good points. I originally went with this method because the data in question is the preprocessed content store and referring to that within the repo directory would mean I would have to move about or have multiple copies of the preprocessed content store. However, you're right, it does make it less reproducible so I'll change it.